src/FSharpx.Collections/Collections.fs

```diff
@@ -234,26 +234,21 @@ module Seq =
 
     /// Extracts from a seq of Choice all the Choice1Of2 elements. All the Choice1Of2 elements are extracted in order.
     let inline choice1s xs =
-        let chooser = function
-                      | Choice1Of2 x -> Some x
-                      | _            -> None
+        let inline chooser a = match a with
```

Comment:
Why the use of `inline` here? Did this help with performance or compilation in some way? I'm not arguing against it; I'm just curious.

Reply:
Was not the whole choide1s already inlined? 
Same question for similar changes below.

Reply:
but not the helper function `chooser`